src/apis/adsense/index.ts

```diff
@@ -20,10 +20,10 @@ export const VERSIONS = {
   'v1.4': adsense_v1_4.Adsense,
 };
 
-export function adsense(version: 'v1_4'): adsense_v1_4.Adsense;
```

Comment:
This is a generated file!  See `/*! THIS FILE IS AUTO-GENERATED */` :) This looks to be a bug with the generator.

Reply:
On it! Thanks